precompile/contracts/deployerallowlist/simulated_test.go

```diff
@@ -217,131 +217,6 @@ func TestDeployerAllowList(t *testing.T) {
 }
 
 func TestIAllowList_Events(t *testing.T) {
```

Comment:
these tests are otherwise identical because it's the same interface -- we can share them and then provide a different backend.